Cmdr/BuiltInCommands/help.lua

```diff
@@ -29,9 +29,27 @@ return {
 			end
 		else
 			local commands = context.Cmdr.Registry:GetCommands()
-
-			for _, cmd in pairs(commands) do
-				context:Reply(("%s - %s"):format(cmd.Name, cmd.Description))
+			local groups = context.Cmdr.Registry:GetGroups()
+			
+			context:Reply([[
+Notes to consider!
+------------------
+. = Me/Self
+* = All/Everyone
+** = Others
+? = Random
+?N = List of N random values	
+]])
```

Comment:
We should move this string to a constant at the top of the file so the indentation isn't as much of a problem.

I'd also separate them with tab characters instead of equal signs.

Replace "Notes to consider!" with "Argument Shorthands"